include/seqan3/io/alignment_file/record.hpp

```diff
@@ -0,0 +1,428 @@
+    ~sam_record() = default; //!< Defaulted.
+
+    //!\brief Inherit std::tuple's and seqan3::record constructors.
+    using base_t::base_t;
+    //!\}
+
+    //!\brief The identifier, usually a string. (SAM Column ID: QNAME)
+    decltype(auto) id() &&
+    {
+        return get_impl(field_constant<seqan3::field::id>{}, static_cast<tuple_base_t &&>(*this));
+    }
+    //!\copydoc seqan3::sam_record::id
+    decltype(auto) id() const &&
+    {
+        return get_impl(field_constant<seqan3::field::id>{}, static_cast<tuple_base_t const &&>(*this));
+    }
+    //!\copydoc seqan3::sam_record::id
+    decltype(auto) id() &
+    {
+        return get_impl(field_constant<seqan3::field::id>{}, static_cast<tuple_base_t &>(*this));
+    }
+    //!\copydoc seqan3::sam_record::id
+    decltype(auto) id() const &
+    {
+        return get_impl(field_constant<seqan3::field::id>{}, static_cast<tuple_base_t const &>(*this));
+    }
+
+    //!\brief The "sequence", usually a range of nucleotides or amino acids. (SAM Column ID: SEQ)
+    decltype(auto) sequence() &&
+    {
+        return get_impl(field_constant<seqan3::field::seq>{}, static_cast<tuple_base_t &&>(*this));
+    }
+    //!\copydoc seqan3::sam_record::sequence
+    decltype(auto) sequence() const &&
+    {
+        return get_impl(field_constant<seqan3::field::seq>{}, static_cast<tuple_base_t const &&>(*this));
+    }
+    //!\copydoc seqan3::sam_record::sequence
+    decltype(auto) sequence() &
+    {
+        return get_impl(field_constant<seqan3::field::seq>{}, static_cast<tuple_base_t &>(*this));
+    }
+    //!\copydoc seqan3::sam_record::sequence
+    decltype(auto) sequence() const &
+    {
+        return get_impl(field_constant<seqan3::field::seq>{}, static_cast<tuple_base_t const &>(*this));
+    }
+
+    //!\brief The qualities, usually in Phred score notation. (SAM Column ID: QUAL)
+    decltype(auto) base_qualities() &&
+    {
+        return get_impl(field_constant<seqan3::field::qual>{}, static_cast<tuple_base_t &&>(*this));
+    }
+    //!\copydoc seqan3::sam_record::base_qualities
+    decltype(auto) base_qualities() const &&
+    {
+        return get_impl(field_constant<seqan3::field::qual>{}, static_cast<tuple_base_t const &&>(*this));
+    }
+    //!\copydoc seqan3::sam_record::base_qualities
+    decltype(auto) base_qualities() &
+    {
+        return get_impl(field_constant<seqan3::field::qual>{}, static_cast<tuple_base_t &>(*this));
+    }
+    //!\copydoc seqan3::sam_record::base_qualities
+    decltype(auto) base_qualities() const &
+    {
+        return get_impl(field_constant<seqan3::field::qual>{}, static_cast<tuple_base_t const &>(*this));
+    }
+
+    /*!\brief Sequence (seqan3::sam_record::sequence) relative start position (0-based), unsigned value. (SAM Column ID: POS)
+    *
+     * \details
+     * The position is the length of the soft-clipping at the start of the seqan3::sam_record::cigar_sequence if a soft-clipping is present, 0 otherwise.
+     */
```

Comment:
```suggestion
    /*!\brief Sequence (seqan3::sam_record::sequence) relative start position (0-based), unsigned value.
     *        (SAM Column ID: POS)
     *
     * \details
     *
     * The position is the length of the soft-clipping at the start of the seqan3::sam_record::cigar_sequence if a 
     * soft-clipping is present and `0` otherwise.
     */
```